### DISCUSSION TITLE
Run ravendb on macos in background

### DISCUSSION THREAD

**Discussion by djordjedjukic** [ravendb/ravendb#14696] [Q&A]
Hi,

Is there a way to run ravendb in the background on macos, as it can be done on windows (windows service)?

**Comment by garayx** [comment]
Hi 

We don't have automated script for that on macos, I have opened an ticket to add it in the future
https://issues.hibernatingrhinos.com/issue/RavenDB-19103/Create-install-daemonsh-alternative-for-macos

**Reply by djordjedjukic** [reply]
That would be cool :)

Thanks

**Comment by tobiaszuercher** [comment]
why not run with docker? thats also in the background ;)

**Reply by djordjedjukic** [reply]
it is ultra slow with docker. 
i.e. when I am importing the same dumb file, with docker it takes ~15 minutes and directly it takes ~1.5 minutes
